europeana-record-id/index.html

```diff
@@ -142,45 +143,35 @@
           const apiKey = extension.parameters.installation.API_KEY;
           const xhr = new XMLHttpRequest();
 
-          xhr.open('GET', 'https://api.europeana.eu/api/v2/record' + recordId + '.json?wskey=' + apiKey);
+
+          xhr.open('GET', 'https://api.europeana.eu/record/search.json?profile=minimal&query=europeana_id%3A%22' + recordId + '%22&rows=1&wskey=' + apiKey);
           xhr.send(null);
           xhr.responseType = 'json';
 
           xhr.onreadystatechange = function () {
             if (xhr.readyState === 4) { // 4 = done
               if (xhr.status === 200) {
-
-                const response = xhr.response.object;
-                let objectTitle;
-
-                // set field values
-                extension.entry.fields.identifier.setValue(recordId);
-                valueRecordID = recordId;
-
-                // TODO: specify a map EDM / other form fields
-                if (response.proxies) {
-                  if (response.proxies[0].dcTitle) {
-                    objectTitle = languageValue(response.proxies[0].dcTitle);
-                    extension.entry.fields.name.setValue(objectTitle);
-                  }
-
-                  if (response.proxies[0].dcCreator) {
-                    // in case of multiple creators, it just picks the first one
-                    extension.entry.fields.creator.setValue(languageValue(response.proxies[0].dcCreator));
+                const response = xhr.response;
+                if (response.itemsCount === 1 && response.items) {
+                  const item = response.items[0];
+                  const objectTitle = languageValue(item.dcTitleLangAware, recordId).substring(0,255);
+                  // set field values
+                  extension.entry.fields.identifier.setValue(recordId);
+                  valueRecordID = recordId;
```

Comment:
Is this `valueRecordID` used somewhere?

Reply:
It's setup outside this block and used in the validation. I think this is to make sure it persists clearing of the form. I don't recall the exact reason it was there, but I didn't want to change it with this PR.